x-pack/solutions/security/plugins/security_solution/public/common/components/user_profiles/use_get_current_user_profile.tsx

```diff
@@ -39,6 +39,7 @@ export const useGetCurrentUserProfile = () => {
     {
       retry: false,
       staleTime: Infinity,
+      refetchOnWindowFocus: false,
```

Comment:
This hook is widely used. Is this change necessary for everybody?

Reply:
It does not give much advantage but one downside i saw was that in case of error, the `failure` notifications keep piling up as you click around or change windows.